contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -842,6 +842,20 @@ CREATE OR REPLACE FUNCTION sys.radians(IN arg1 NUMERIC)
 RETURNS numeric  AS 'babelfishpg_tsql','numeric_radians' LANGUAGE C STRICT IMMUTABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION sys.radians(NUMERIC) TO PUBLIC;
 
+CREATE OR REPLACE FUNCTION Database_principal_id(user_name TEXT)
+RETURNS OID AS $$
+BEGIN
+    RETURN sys.user_id(LOWER(user_name));
```

Comment:
from my POV, database_principal_id is equivalent to user_id. 
Could you please point out what is the additional behavior you would like to add on top of it? 
Isn't case sensitivity handled internally by user_id implementation?

Reply:
Yes removed the LOWER